lib/matplotlib/axes/_axes.py

```diff
@@ -8226,6 +8227,21 @@ def violinplot(self, dataset, positions=None, vert=True, widths=0.5,
           its only parameter and return a scalar. If None (default), 'scott'
           is used.
 
+        facecolor : color or list of colors or None; see :ref:`colors_def`
+          If provided, will set the face color(s) of the violin plots. The alpha
+          value is automatically set to 0.3.
```

Comment:
I'm trying to figure out what the semantics _should_ be to remain consistent with current behavior... It feels wrong to override the alpha value of a user provided color...

Though I will also note that the `0.3` alpha is actually applied to the patch object as a whole, not its color values...

Makes me wonder if the "better" solution would be to just pass `**kwargs` to lower level artists (But I think that gets complicated quickly because some of them take the "edgecolor" as "color", for instance..., and other values would have to be filtered out likely...)

Reply:
That's a good point. I think that if the user doesn't specify the alpha, it makes sense to set it to 0.3. That just looks good, and honestly I'd prefer that as someone who wants to use the violinplot method with these new features. 

I added a commit that changes the behavior: 
If the user specifies the alpha, then that is the alpha that is used. If the user doesn't specify the alpha, then it defaults to 0.3.